aito/sdk/aito_client.py

```diff
@@ -270,64 +270,68 @@ def optimize_table(self, table_name):
             LOG.error(f'failed to optimize: {e}')
         LOG.info(f'table {table_name} optimized')
 
-    def upload_entries(self, table_name: str, entries: List[Dict], optimize_on_finished: bool = True):
-        """populate a list of entries to a table API doc
-
-        `API doc <https://aito.ai/docs/api/#post-api-v1-data-table-batch>`__
-        if the list contains more than 1000 entries, upload the entries by batches of 1000
-
-        :param table_name: the name of the table
-        :type table_name: str
-        :param entries: list of the table entries
-        :type entries: List[Dict]
-        :param optimize_on_finished: `optimize https://aito.ai/docs/api/#post-api-v1-data-table-optimize`__ the table on finished, defaults to True
-        :type optimize_on_finished: bool
-        """
-        if len(entries) > 1000:
-            self.upload_entries_by_batches(table_name, entries, optimize_on_finished)
-        else:
-            LOG.debug(f'uploading {len(entries)} to table `{table_name}`...')
-            self.request('POST', f"/api/v1/data/{table_name}/batch", entries)
-            LOG.info(f'uploaded {len(entries)} entries to table `{table_name}`')
-            if optimize_on_finished:
-                self.optimize_table(table_name)
-
-    def upload_entries_by_batches(
+    def upload_entries(
             self,
             table_name: str,
-            entries: List[Dict],
+            entries: Generator,
             batch_size: int = 1000,
             optimize_on_finished: bool = True
     ):
         """populate a list of table entries by batches of batch_size
-
         :param table_name: the name of the table
         :type table_name: str
         :param entries: list of the table entries
-        :type entries: List[Dict]
+        :type entries: Generator
         :param batch_size: the batch size, defaults to 1000
         :type batch_size: int, optional
         :param optimize_on_finished: `optimize https://aito.ai/docs/api/#post-api-v1-data-table-optimize`__ the table on finished, defaults to True
         :type optimize_on_finished: bool
         """
-        LOG.debug(f'uploading {len(entries)} entries to table `{table_name}` with batch size of {batch_size}...')
+        LOG.info(f'uploading entries to table `{table_name}` with batch size of {batch_size}...')
```

Comment:
This should be debug instead of info. We log info after the process finished

Reply:
I modified this as I think it's good to inform the user about starting the upload